packages/labs/rollup-plugin-minify-html-literals/package.json

```diff
@@ -53,11 +53,13 @@
       ]
     },
     "test": {
-      "command": "nyc mocha -r ts-node/register -r source-map-support/register test/*.js",
+      "command": "nyc mocha -r source-map-support/register test/*.js",
       "dependencies": [
         "build"
       ],
-      "files": [],
+      "files": [
+        "package.json"
```

Comment:
package.json doesn't need to be listed in files: https://docs.npmjs.com/cli/v10/configuring-npm/package-json#files

Reply:
this was wireit's `files` so changing the `nyc` config would trigger a cache clear

i dropped `nyc` in the end but we do still have a c8 config in the same place. upto you, it was just a convivence thing while i was running locally

Reply:
Ah, ok. I misread which files property this was.

Since you dropped nyc, do you still need the `"nyc"` section below?

Reply:
that part is done in https://github.com/lit/lit/pull/4737